src/pages/index.js

```diff
@@ -18,6 +18,8 @@ const imageUrlInput = document.querySelector("#image-url-input");
 const profileTitleInput = document.querySelector("#profile-title-input");
 const profileEditModal = document.querySelector("#profile-edit-modal");
 const profileEditForm = profileEditModal.querySelector("#profile-modal-form");
+const avatarEditModal = document.querySelector("#update-avatar-modal");
+const avatarEditForm = avatarEditModal.querySelector("#update-avatar-form");
```

Comment:
**Could be improved**

We can get any form using their **unique** attribute `name` or `id` from `document.forms`

```js

const profileForm = document.forms["profile-form"];
const cardForm = document.forms["card-form"];

```

You need to add `name` or `id` first in `index.html` to achieve that.

More info here https://developer.mozilla.org/en-US/docs/Web/API/Document/forms